server/render-component-page.js

```diff
@@ -1,8 +1,13 @@
 /**
  * generate a URL's static HTML 
+ * @param {object} generator the server page generator
+ * @param {string} location the URL for this request
+ * @param {object} res express.js response object 
+ * @param {string} locale the locale/language for this page
+ * @returns {undefined}
  */
-module.exports = function renderComponentPage(serverBundle, location, res, locale) {
-  serverBundle.generate(location, {locale: locale}, function(err, location, title, html) {
+module.exports = function renderComponentPage(generator, location, res, locale) {
+  generator.generate(location, {locale: locale}, function(err, _location, title, html) {
```

Comment:
I'd prefer not to - the javadoc rule made it obvious that this argument, at least for this function, not its called, should talk about generators, not "whatever that generator was loaded as".